ChunkyDocs/docs/reference/user_interface/chunky/render_controls/sky_and_fog.md

```diff
@@ -0,0 +1,191 @@
+# Render Controls - Sky & Fog
+
+The <samp>Sky & Fog</samp> tab contains controls for the appearance of the sky in the scene, and for fog.
+
+<div class="figure" id="figure-1">
+  <p class="figure">Figure 1: The <samp>Sky & Fog</samp> tab</p>
+  <div class="figureimgcontainer">
+    <a href="../../../../../img/reference/user_interface/chunky/render_controls/sky_and_fog/sky_and_fog_tab.png">
+      <img class="figure" src="../../../../../img/reference/user_interface/chunky/render_controls/sky_and_fog/sky_and_fog_tab.png" alt="Sky & Fog tab">
+    </a>
+  </div>
+</div>
+
+## Sky Mode Settings
+
+- <samp>Sky mode</samp>: Dropdown menu to select the type of sky to be used for the scene.
```

Comment:
I believe there was an idea to remove some indents and to add more sub-headings. This part of the article certainly looks like it could do with this treatment.